includes/class-ckwc-order.php

```diff
@@ -150,6 +155,36 @@ public function maybe_subscribe_customer( $order_id, $status_old = 'new', $statu
 			$subscriptions[] = $resource_id;
 		}
 
+		// Get coupon-specific subscription settings.
+		foreach ( $order->get_coupon_codes() as $coupon_code ) {
+			// Get the WC_Coupon object.
+			$coupon = new WC_Coupon( $coupon_code );
+
+			// Get the Form, Tag or Sequence for this Coupon.
+			$resource_id = get_post_meta( $coupon->get_id(), 'ckwc_subscription', true );
+
+			/**
```

Comment:
Was this comment intentional?

Reply:
Yes :) this fetches the setting on the coupon to determine which form, tag or sequence (if any) to subscribe the customer to. The comment follows what we do for products, which can be found [here](https://github.com/ConvertKit/convertkit-woocommerce/pull/141/files/7ef20314d75cc0e7028566cb0b53b43b252d2bef#diff-682d8bc983417c0b52a7e93ba02c49b9db5e09fc2e6067b9fd2ef515fe6d9defR133)